inference-engine/tests/functional/plugin/cpu/single_layer_tests/concat.cpp

```diff
@@ -118,28 +118,28 @@ INSTANTIATE_TEST_SUITE_P(smoke_Concat4D_CPU_Block8, ConcatLayerCPUTest,
 
 INSTANTIATE_TEST_SUITE_P(smoke_Concat4D_CPU_Block16inPlace, ConcatLayerCPUTest,
                         ::testing::Combine(
-                                ::testing::Values(1),
-                                ::testing::Values(std::vector<std::vector<size_t>>{{2, 16, 3, 5},
-                                                                                   {2, 32, 3, 5}}),
+                                ::testing::Values(0, 1),
+                                ::testing::Values(std::vector<std::vector<size_t>>{{1, 32, 3, 5},
+                                                                                   {1, 32, 3, 5}}),
```

Comment:
There are no tests on concatenation by the batch dimension where n != 1, I think it would be great to add such tests.

Reply:
I'm agree. [Added](https://github.com/openvinotoolkit/openvino/blob/0f068a3589dc0b58d84e8efc61d2a1c3e8d1b776/inference-engine/tests/functional/plugin/cpu/single_layer_tests/concat.cpp#L192). Thank you!